aya/src/programs/tc.rs

```diff
@@ -89,6 +97,12 @@ pub enum TcError {
     /// the clsact qdisc is already attached
     #[error("the clsact qdisc is already attached")]
     AlreadyAttached,
+    /// tcx links can only be attached to ingress or egress
+    #[error("tcx links can only be attached to ingress or egress")]
```

Comment:
this phrasing is weird at a distance. where you emit the error you just know you're holding a `Custom`. Should we include the value in this error?

Reply:
Sure, done and fixed up error